modules/authentication/proc-enabling-authentication-with-rhsso.adoc

```diff
@@ -0,0 +1,172 @@
+[id="enabling-authentication-with-rhsso"]
+= Enabling authentication with Red Hat Single-Sign On (RHSSO)
+
+To authenticate users with Red Hat SSO (RHSSO), enable the OpenID Connect (OIDC) authentication provider in {product}.
+
+
+.Prerequisites
+* You link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/{product-version}/html/administration_guide_for_red_hat_developer_hub/assembly-add-custom-app-file-openshift_admin-rhdh[added a custom {product-short} application configuration], and have sufficient permissions to modify it.
+* You have sufficient permissions in RHSSO to create and manage a realm.
+
+.Procedure
+. To allow Developer Hub to authenticate with RHSSO, complete the steps in RHSSO, to link:https://docs.redhat.com/en/documentation/red_hat_single_sign-on/7.6/html-single/getting_started_guide/index#realms-apps_[create a realm and a user] and link:https://docs.redhat.com/en/documentation/red_hat_single_sign-on/7.6/html-single/getting_started_guide/index#registering-app_[register the {product-short} application]:
+
+.. Use an existing realm, or link:https://docs.redhat.com/en/documentation/red_hat_single_sign-on/7.6/html-single/getting_started_guide/index#create-realm_[create a realm], with a distinctive **Name** such as __<my_realm>__.
+Save the value for the next step:
+* **RHSSO realm base URL**, such as: __<your_rhsso_URL>__/auth/realms/__<your_realm>__.
+
+.. To register your {product-short} in RHSSO, in this realm, link:https://docs.redhat.com/en/documentation/red_hat_single_sign-on/7.6/html-single/getting_started_guide/index#registering-app_[create a Client ID], with:
+... **Client ID**: A distinctive client ID, such as __<{product-very-short}>__.
+... **Valid redirect URIs**: Set to the OIDC handler URL: `https://__<RHDH_URL>__/api/auth/oidc/handler/frame`.
+... Navigate to the **Credentials** tab and copy the **Client secret**.
+... Save the values for the next step:
+* **Client ID**
+* **Client Secret**
+
+.. To prepare for the verification steps, in the same realm, get the credential information for an existing user or link:https://docs.redhat.com/en/documentation/red_hat_single_sign-on/7.6/html-single/getting_started_guide/index#create-user_[create a user]. Save the user credential information for the verification steps.
+
+. To add your RHSSO credentials in your Developer Hub secrets, edit your Developer Hub secrets, such as `secrets-rhdh`, and add the following key/value pairs:
++
+`AUTH_OIDC_CLIENT_ID`:: Enter the saved **Client ID**.
+`AUTH_OIDC_CLIENT_SECRET`:: Enter the saved **Client Secret**.
+`AUTH_OIDC_METADATA_URL`:: Enter the saved **RHSSO realm base URL**.
+
+. To set up the RHSSO authentication provider in your {product-short} custom configuration, edit your custom Developer Hub config map, such as `app-config-rhdh`, and add following lines to the `app-config-rhdh.yaml` content:
++
+--
+.`app-config-rhdh.yaml` fragment with mandatory fields to enable authentication with RHSSO
```

Comment:
```suggestion
.Example `app-config-rhdh.yaml` fragment with mandatory fields to enable authentication with RHSSO
```

Reply:
For minimalism, fragment is enough, example would be a répétition.